### ISSUE TITLE
Samsung Galaxy A13, none of the apks work

### ISSUE THREAD

**Issue by mapma1** [lrorpilla/jidoujisho#146]
Hi,
I'm not a very technical person but the instructions on the release say to try each type (arm64, armeabi, x86_64) to see which is compatible. I downloaded everything and tried them one by one and all of them say they aren't compatible when I try to open the apk. 
Couldn't find any similar issues when doing a cursory search in the closed issues.
Appreciated in advance.

**Comment by lrorpilla** [comment]
Can you send a recording of you installing the application? How much free space do you have on the device?

**Comment by mapma1** [comment]
Hi,
When I was about to go to sleep I attempted to install again the version I assumed wasn't standalone and was an old patch "[jidoujisho_yuuna_2_3_0-armeabi-hotfix.apk](https://github.com/lrorpilla/jidoujisho/releases/download/2.3/jidoujisho_yuuna_2_3_0-armeabi-hotfix.apk)" and it ended up working. 

If the version I linked is good then my issue is resolved but if me being unable to download any of the other versions that were armeabi then let me know and I will record the application. I also have enough free space since the phone is newly purchased.
